livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -64,7 +64,7 @@ public void open() {
 
   @Override
   public void close() {
-    livyHelper.closeSession(userSessionMap, "spark");
+    livyHelper.closeSession(userSessionMap, "LivySparkSQLInterpreter");
```

Comment:
should it call `LivySparkSQLInterpreter.class.getName()` instead of hardcoding the class name?